daemon/main.rs

```diff
@@ -49,6 +52,7 @@ async fn run(config: CliArgs) -> Result<()> {
         temp_dir: Some(lassie_temp_dir),
         // Listen on an ephemeral port selected by the operating system
         port: 0,
+        access_token: Some(generate_lassie_access_token()),
```

Comment:
I didn't test this change at all. It's difficult to test it since the access token is not visible to Zinnia modules.